.github/workflows/bedrock.yml

```diff
@@ -0,0 +1,90 @@
+name: Bedrock Test Suite
+on:
+  push:
+    branches: # this ignores tag pushes, and only looks at branches.
+      - '**'
+  release:
+    types: [prereleased]
```

Comment:
This is because we only trigger pre-releases here: https://github.com/Expensify/Bedrock/releases